visual-testing-app/src/components/public-page-form-wide/public-page-form-wide.visualroute.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { PublicPageLayout } from '@commercetools-frontend/application-components';
+import DoorsClosedSvg from '@commercetools-frontend/assets/images/project-not-initialized.svg';
+// @ts-ignore
+import Link from '@commercetools-uikit/link';
+import Spacings from '@commercetools-uikit/spacings';
+import { customProperties } from '@commercetools-uikit/design-system';
+import Text from '@commercetools-uikit/text';
+import { Suite, Spec } from '../../test-utils';
+
+export const routePath = '/public-page-form-wide';
+
+const LegalMessage = () => (
+  <Text.Body tone="inverted">
+    Read our{' '}
+    <Link isExternal={true} to="https://commercetools.com/privacy#suppliers">
```

Comment:
Should these be env variables maybe or is there little value on doing so?

Reply:
I don't think there is any value to it for the moment, nor do I think it should be flexible derived from configuration yet.

Reply:
I think it's fine to have them hardcoded. In fact they don't even need to be "real links". It's just so that the component is rendered as a link, as so the snapshot can catch unwanted changes.